TACC/Frontera-Portal
# task/FP-777 - project membership step port from CEP v1

## Overview: ##

Implement Project Membership onboarding step. (Checks for membership in TAS project, then allows a user to send a request to be added to project.)

## PR Status: ##


## Related Jira tickets: ##

* [FP-777](https://jira.tacc.utexas.edu/browse/FP-777)

## Summary of Changes: ##

- Adds project_membership_step
- default settings.py includes this step for a test project (this should not go into prod settings)

## Testing Steps: ##
1. Make sure you are not part of the WMA-Stampede2 project (contact jchuah)
2. Delete previous setup events and reset your setupComplete flag
3. Log in. Onboarding should prompt you to click Request Project Access
4. Click the Request Project Access link
5. Set your user is_staff flag to True and refresh the onboarding page
6. Click the "Grant Project Access" link
7. Check to make sure the generated Rt ticket is closed and that you have been added to WMA-Stampede2

## UI Photos:

## Notes: ##

Settings need to have this disabled before put in prod
